src/domain/synthetics/markets/useMarketTokensAPR.ts

```diff
@@ -39,59 +41,27 @@ function useMarketAddresses(chainId: number) {
 }
 
 function useIncentivesBonusApr(chainId: number): MarketTokensAPRData {
-  const rawIncentivesStats = useIncentiveStats(chainId);
-  const { tokensData } = useTokensDataRequest(chainId);
   const marketAddresses = useMarketAddresses(chainId);
-  const { marketsInfoData } = useMarketsInfoRequest(chainId);
 
   return useMemo(() => {
-    let arbTokenAddress: null | string = null;
-    try {
-      arbTokenAddress = getTokenBySymbol(chainId, "ARB").address;
-    } catch (err) {
-      // ignore
-    }
-    let arbTokenPrice = BigNumber.from(0);
-
-    if (arbTokenAddress && tokensData) {
-      arbTokenPrice = tokensData[arbTokenAddress]?.prices?.minPrice ?? BigNumber.from(0);
-    }
-
-    const shouldCalcBonusApr =
-      arbTokenPrice.gt(0) && (chainId === ARBITRUM || chainId === ARBITRUM_GOERLI) && rawIncentivesStats?.lp.isActive;
-
     return marketAddresses.reduce((acc, marketAddress) => {
-      if (!shouldCalcBonusApr || !rawIncentivesStats || !rawIncentivesStats.lp.isActive)
-        return { ...acc, [marketAddress]: BigNumber.from(0) };
-
-      const arbTokensAmount = BigNumber.from(rawIncentivesStats.lp.rewardsPerMarket[marketAddress] ?? 0);
-      const yearMultiplier = Math.floor((365 * 24 * 60 * 60) / rawIncentivesStats.lp.period);
-      const poolValue = getByKey(marketsInfoData, marketAddress)?.poolValueMin;
-      let incentivesApr = BigNumber.from(0);
-
-      if (poolValue?.gt(0)) {
-        incentivesApr = arbTokensAmount
-          .mul(arbTokenPrice)
-          .div(poolValue)
-          .mul(yearMultiplier)
-          .div(expandDecimals(1, 14));
-      }
-
       return {
         ...acc,
-        [marketAddress]: incentivesApr,
```

Comment:
wondering is it necessary to remove this code? we will probably have another round of incentives in next couple of months

Reply:
fixed